modules/machine-user-provisioned-limitations.adoc

```diff
@@ -1,13 +1,17 @@
 // Module included in the following assemblies:
 //
-// * machine_management/applying_autoscaling.adoc
 // * machine_management/creating-infrastructure-machinesets.adoc
-// * machine_management/creating-machinesets.adoc
```

Comment:
This assembly does not exist (and perhaps never did?). Replaced with provider-specific refs.